Services/UI/OJS.Services.Ui.Models/Contests/LecturerInContestServiceModel.cs

```diff
@@ -0,0 +1,20 @@
+namespace OJS.Services.Ui.Models.Contests;
+
+using OJS.Data.Models;
+using AutoMapper;
+using SoftUni.AutoMapper.Infrastructure.Models;
+
+public class LecturerInContestServiceModel : IMapExplicitly
```

Comment:
Would it work with IMapFrom and IMapTo instead of IMapExplicitly, so that you get rid of the RegisterMappings method ?

Reply:
No, because the essential thing here is the ignoring of all other properties besides the two ids.